src/Symfony/Component/EventDispatcher/DependencyInjection/RegisterListenersPass.php

```diff
@@ -195,7 +178,7 @@ private function getEventFromTypeDeclaration(ContainerBuilder $container, string
             || $type->isBuiltin()
             || Event::class === ($name = $type->getName())
         ) {
-            throw new InvalidArgumentException(sprintf('Service "%s" must define the "event" attribute on "%s" tags.', $id, $this->listenerTag));
+            throw new InvalidArgumentException(sprintf('Service "kernel.event_listener" must define the "event" attribute on "%s" tags.', $id));
```

Comment:
This change is not correct. See the test failures on Travis.

Reply:
I've seen this already 🙈